l1-contracts-foundry/script/Script.sol

```diff
@@ -5,9 +5,8 @@ import {Script as ForgeScript} from "forge-std/Script.sol";
 
 contract Script is ForgeScript {
     uint256 chainId;
-    string network;
 
     function isNetworkLocal() internal view returns (bool) {
-        return keccak256(abi.encodePacked(network)) == keccak256(abi.encodePacked("local"));
+        return chainId == 31337;
```

Comment:
It's not true, we use 9 as localhost.

Reply:
BTW, do we need `isNetworkLocal` anywhere except create2 and multicall3? could we just check that the bytecode is not empty ?

Reply:
That's a better solution indeed. Fixed in b8141f75039363880f2cda8a6c312e68eeb60b3d